### PR TITLE
Fix suggester

### PR DESCRIPTION
**Pull request by kirti-d**
provide fixes for errors by searching stackoverflow using google custom search API

### CODE REVIEW

File: src/main/java/com/google/error_analyzer/data/ErrorFixes.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+public class ErrorFixes{
+    
+    private static final Logger logger = LogManager.getLogger(ErrorFixes.class);
+    public String findFixes(String searchQuery) {
+    
+        try{
+            String searchEngine = "cx"; //Your search engine
+
+            //Instance Customsearch
+            Customsearch customeSearch =
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/data/ErrorFixes.java:40]: customSearch

File: src/main/java/com/google/error_analyzer/data/ErrorFixes.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class ErrorFixes{
+    
+    private static final Logger logger = LogManager.getLogger(ErrorFixes.class);
+    public String findFixes(String searchQuery) {
+    
+        try{
+            String searchEngine = "cx"; //Your search engine
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/data/ErrorFixes.java:37]: What is "cx"?
Should we create a constant?

File: src/main/java/com/google/error_analyzer/data/ErrorFixes.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                                                    .setCx(searchEngine); 
+            
+            //Execute search
+            Search result = list.execute();
+            
+            // return url in a tag 
+            if (result.getItems()!=null){
+                Result stackoverflowResult= result.getItems().get(0);
+                String fix= stackoverflowResult.getLink();
+                fix=" <a href="+fix+"> FIX </a>";
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/data/ErrorFixes.java:58]: create a constant for string part and use String.format()

File: src/main/java/com/google/error_analyzer/data/ErrorFixes.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/**Copyright 2019 Google LLC
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/data/ErrorFixes.java:1]: Test file for ErrorFixes.java?

File: src/main/java/com/google/error_analyzer/data/ErrorFixes.java (hunk 2 above)
Comment (kirti-d) [reply src/main/java/com/google/error_analyzer/data/ErrorFixes.java:37]: we need to replace cx with search engine id(can't put it on git)

**Comment by nishantnahata** [comment]
Move the servlet tests to servlet/ directory

**Pull request merged**